slave/buildslave/commands/registry.py

```diff
@@ -21,16 +21,7 @@
     "uploadFile": "buildslave.commands.transfer.SlaveFileUploadCommand",
     "uploadDirectory": "buildslave.commands.transfer.SlaveDirectoryUploadCommand",
     "downloadFile": "buildslave.commands.transfer.SlaveFileDownloadCommand",
-    "svn": "buildslave.commands.svn.SVN",
```

Comment:
What do you think about grouping all these commands at the end of the registry and changing corresponding class to something that produces an error?